apps/system/js/notifications.js

```diff
@@ -84,7 +84,7 @@ var NotificationScreen = {
     // screen
     this.externalNotificationsCount = 0;
     this.unreadNotifications = [];
-
+    this.availableNotification = [];
```

Comment:
I'd like a comment here explaining what this property is for.  And shouldn't it be plural rather than singular? Do unreadNotifications matter anymore, or can you delete that property? Or can you just use unreadNotifications instead of availableNotifications?